test/e2e/traceflow_test.go

```diff
@@ -23,6 +23,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/kr/pretty"
```

Comment:
This line should be in line 27,  there is no need to have a new import section.

Reply:
Ok